docs/workflow/code-review.md

```diff
@@ -0,0 +1,103 @@
+# Code review
+
+- [How to submit code for review](#how-to-submit-code-for-review)
+- [How to do a code review](#how-to-do-a-code-review)
+
+## How to submit code for review
+
+1. [Pick an issue to work on](./issue-management.md#how-to-pick-an-issue)
+
+2. Check out a new branch from the latest `main` branch
+    ```bash
+    git checkout -b [branch-name]
+    ```
+
+3. Solve the issue
+
+    > If you've made a lot of changes (>300 LOC), split the changes into multiple PRs - TBA how
+
+4. Push the changes and open a PR
+
+    4.1. Add `Closes [issue-link]` to the description
```

Comment:
that's a github feature on how to link PRs to issues https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword